cpp/texture.cpp

```diff
@@ -36,6 +37,9 @@ GLint base_texture, mask_texture, base_coord, mask_coord, show_mask;
 Texture::Texture(int width, int height, void *data)
 	:
 	use_metafile{false} {
+		
+	assert(glGenBuffers != nullptr && "gl not initialized properly");
```

Comment:
And ...? I think the benefit of immediately knowing what the problem is outweighs the few bytes of diskspace and is a common pattern when using assert. The string wont be in the binary.